troubleshoot/observability/troubleshoot-mapping-issues.md

```diff
@@ -2,6 +2,10 @@
 navigation_title: "Mapping issues"
 mapped_pages:
   - https://www.elastic.co/guide/en/observability/current/troubleshoot-uptime-mapping-issues.html
+applies_to:
+  stack: all
+  serverless:
+    observability: all
```

Comment:
Uptime was deprecated in 8.15.0, but will continue to be available through 9.x so we can't remove the docs yet. 

```suggestion
applies_to:
  stack: deprecated 8.15.0
```